### PR TITLE
[6081] My Activity dashlet allowing operation Reject on already rejected item not pending for approval

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/6081

### CODE REVIEW

File: ui/app/src/components/MyRecentActivityDashlet/MyRecentActivityDashlet.tsx [hunk 1]
```diff
@@ -179,12 +180,13 @@ export function MyRecentActivityDashlet(props: MyRecentActivityDashletProps) {
...
     const hostToHost$ = getHostToHostBus();
     const subscription = hostToHost$.pipe(filter((e) => events.includes(e.type))).subscribe(({ type, payload }) => {
+      dispatch(fetchSandboxItems({ paths: selectedPaths }));
       loadPage(getCurrentPage(offset, limit), true);
     });
     return () => {
       subscription.unsubscribe();
     };
-  }, [limit, offset, loadPage, username]);
+  }, [limit, offset, loadPage, username, dispatch, selectedPaths]);
```
Comment (rart) [inline ui/app/src/components/MyRecentActivityDashlet/MyRecentActivityDashlet.tsx:189]: @jvega190 Did the linter not complain about this or you just missed the warning?

**Review by rart — APPROVED** [review]

File: ui/app/src/components/MyRecentActivityDashlet/MyRecentActivityDashlet.tsx (hunk 1 above)
Comment (jvega190) [reply ui/app/src/components/MyRecentActivityDashlet/MyRecentActivityDashlet.tsx:189]: I saw no warnings, do you mean about `fetchSandboxItems`?

Comment (rart) [reply ui/app/src/components/MyRecentActivityDashlet/MyRecentActivityDashlet.tsx:189]: The warning about the effect not having all dependencies listed (the cause of the bug)

Comment (rart) [reply ui/app/src/components/MyRecentActivityDashlet/MyRecentActivityDashlet.tsx:189]: Never mind I just noticed that was a new line too

**Pull request merged**